content/post/autumn-tests-and-new-features.md

```diff
@@ -0,0 +1,55 @@
+
+We received some very useful feedback and could already correct bugs and undertake enhancements. Other improvements in the "waiting queue" are documented in our public available [Github kanban](https://github.com/orgs/rero/projects/4).
+
+In the meantime, we have also been working on the implementation of missing functionalities. A new version has been published on the [public test portal](https://sonar.test.rero.ch/) by the end of November.
+
+### New features
+
+4 important new features have been deployed.
+
+**1. OAI-PMH**
+
+This protocol is a *must* in the small world of institutional repositories, as it enables to harvest the metadata in order to reference it in another search portal. The most common use case is probably to integrate an institutional repository into the institution's library discovery tool.
+
+In SONAR, one set is automatically generated for each member institution. The export format is currently Dublin Core and its 10 base elements. We tried to stay as close as possible to the [OpenAIRE guidelines, v3](https://guidelines.readthedocs.io/en/latest/literature/application_profile.html). We are already thinking about adding other more granular export formats.
+
+To have a look, try to add `/oai2d?verb=ListRecords&metadataPrefix=oai_dc` to the base URL of the public test portal: [link to try!](https://sonar.test.rero.ch/oai2d?verb=ListRecords&metadataPrefix=oai_dc)
+
+**2. Licence**
+
+It is now possible to specify a licence, from a closed list that was intentionally kept generic so that it can be understood and used by the majority of people.
+
+<img class="image fit" src="/images/licence.png" alt="Licence choice in SONAR" title="Licence choice in SONAR" >
+
+**3. File management**
+
+Until now, it was possible to deposit a main file and to attach additional annexes. From now, a manager can see the file list and modify for each file its position, its displayed label and its access rights. The latter can be defined precisely between open access, embargo, restricted access. It is also possible to enable access only from the organisation's IP addresses.
+
+**4. Support of all file formats**
+
+SONAR is now able to receive any kind of file formats. A preview is available for PDF and some images formats. The other file types must be downloaded to be visualised.
+
+### Small corrections
+
+We could already correct some small things pointed out by the test institutions, for instance:
+
+* ORCID supports X as last character.
+* Bookmarks are better displayed in PDF previews.
+
+### Next to come
+
+The integration of **project and funding information** in the document metadata is under development. This will enable to navigate in the SONAR interface between documents through research projects.
```

Comment:
```suggestion
The integration of **research project and funding information** in the document's metadata is under development. This will enable to navigate in the SONAR interface between documents through research projects.
```